tests/unit/interceptors/urllib3_test.py

```diff
@@ -6,12 +6,12 @@
 
 @pook.on
 def assert_chunked_response(input_data, expected):
-    (pook.get('httpbin.org/foo')
+    (pook.get('https://httpbin.org/foo')
```

Comment:
Just to clarify, are these changes necessary for the fix as well?

Reply:
They're necessary for v2+ support, yes. There is `DeprecationWarning` from urllib3 without that (see my commit messages)